server/src/admin/commands/admin-commands/CleanUpTeam.ts

```diff
@@ -0,0 +1,29 @@
+import { RegisteredUserAppCommand } from "../DiscordAppCommand.js";
```

Comment:
Every command filename is "DoSomethingCommand.ts" - should rename this file appropriately